### DISCUSSION TITLE
Setup location help needed

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#976] [General]
Hey guys, bit by bit my setup is getting here (the battery cells and the Victron MPPT) and I started thinking about how to put this all into a complete setup.

Option A: outside. Unfornutately not an option, as in winter we have -10 (usually) and a day or two maybe -15..-20. Anyhow, the Lifepo4 won't survive.

Option B: put it all together in the house (Hoymiles, MPPT, Lifepo4 battery with JKBMS). The only place available I have in the house is above my "natural gas boiler", a shelf above it. But is it safe to have the battery in there? As I saw from internet  Lifepo4 is considered quite safe, but still - there's gas nearby.

What would you do in this situation?

**Comment by swingstate** [comment]
I have everything outside in a DIY isolated and heated box. 

If you have a house I would drill a hole for the MC4 cables and put everything inside, above the boiler as long as temperature above the boiler are moderate.

**Reply by greymda** [reply]
the hole already exists, it's for ventilation of the cabinet where the gas boiler lives. it's just mentally i'm a bit concern of the "gas+battery" combo in the same cabinet (although the battery will be on a separate shelf in there).
temperature is always ±25-30C.

**Reply by swingstate** [reply]
Good Temperatures. My US5000 are at 25-33 in that box.

Make sure you have WiFi in that corner.

**Reply by schlimmchen** [reply]
If you had an ignitable gas+air mixture in the cabinet, you would have blown the house up already. I would also rather consider the temperatures, not that there is a gas-driven heater in the vicinity. If they are under 30°C constantly I wouldn't mind.

**Reply by greymda** [reply]
to be honest my concern was more not about the gas but how safe the battery is.

**Reply by stanpete78** [reply]
I've watched countless LiFePo4 destruction tests and I've never seen one catching fire. I would not worry about it.
